loader/src/ui/nodes/TextAreaV2.cpp

```diff
@@ -0,0 +1,377 @@
+#include <Geode/ui/TextAreaV2.hpp>
+#include <Geode/DefaultInclude.hpp>
+
+using namespace geode::prelude;
+
+#define IMPL_GETTER(type, name, methodName) \
```

Comment:
We generally don't use macros like this, but it's fine here ig

Reply:
Thank god, writing them all out would be painful